### ISSUE TITLE
using the Poisson solution to distinguish interior and exterior points

### ISSUE THREAD

**Issue by abby-normal2** [mkazhdan/PoissonRecon#191]
greetings professor.  apologies if this question is naive ... is it possible using the final set of basis functions
to distinguish points interior and exterior to the triangulated surface?

**Comment by mkazhdan** [comment]
Yes. You can sample the implicit function at arbitrary locations and test if the value is bigger/smaller than the iso-value (0.5).
